code/datums/uplink_item.dm

```diff
@@ -74,23 +74,23 @@ var/list/uplink_items = list()
 		desc = replacetext(initial(temp.desc), "\n", "<br>")
 	return desc
 
-/datum/uplink_item/proc/buy(var/obj/item/device/uplink/hidden/U, var/mob/user)
+/datum/uplink_item/proc/buy(obj/item/device/uplink/hidden/U, mob/user)
 	..()
 
 	if(!istype(U))
-		return 0
+		return FALSE
```

Comment:
Yeah, we're not going down the Baystation route of "no correcting of poor styling changes; save it explicitly for a styling change PR"; I'm thankful he's even addressing this in all of his PRs to begin with.